api/src/basic.rs

```diff
@@ -60,6 +65,56 @@ impl BasicApi {
         Html(OPEN_API_HTML.to_string())
     }
 
+    /// Show some basic info of the node.
+    #[oai(
+        path = "/info",
+        method = "get",
+        operation_id = "info",
+        tag = "ApiTags::General"
+    )]
+    async fn info(&self) -> Json<HashMap<String, String>> {
+        let mut info = HashMap::new();
+        info.insert(
+            "Bootstrapping Mode".to_string(),
+            format!(
+                "{:?}",
+                self.context
+                    .node_config
+                    .state_sync
+                    .state_sync_driver
+                    .bootstrapping_mode
+            ),
+        );
+        info.insert(
+            "Continuous Syncing Mode".to_string(),
+            format!(
+                "{:?}",
+                self.context
+                    .node_config
+                    .state_sync
+                    .state_sync_driver
+                    .continuous_syncing_mode
+            ),
+        );
+        info.insert(
+            "New Storage Format".to_string(),
+            format!(
+                "{:?}",
+                self.context
+                    .node_config
+                    .storage
+                    .rocksdb_configs
+                    .enable_storage_sharding
+            ),
+        );
+        info.insert(
+            "Internal Indexer Config".to_string(),
+            format!("{:?}", self.context.node_config.indexer_db_config),
+        );
+
+        Json(info)
```

Comment:
I suppose this is intended only for human consumption, hence the keys like "Internal Indexer Config" rather than `internal_indexer_config`?

Reply:
Yes that's what I'm expecting. Changed to snake_case anyway.